components/board/numato/board.go

```diff
@@ -409,8 +410,7 @@ func connect(ctx context.Context, name resource.Name, conf *Config, logger loggi
 
 	b.lines = make(chan string)
 
-	b.activeBackgroundWorkers.Add(1)
-	utils.ManagedGo(b.readThread, b.activeBackgroundWorkers.Done)
+	b.workers = rdkutils.NewStoppableWorkers(b.readThread)
```

Comment:
would you be able to have more than one `stoppableworker` in `b.workers` ?

Reply:
Yes, `NewStoppableWorkers` uses a variadic argument:
```go
// Create 3 background goroutines
b.workers = rdkutils.NewStoppableWorkers(b.readThread, b.spinDiscoBall, b.petPuppies)
...
b.workers.Stop() // Block until all 3 goroutines shut down
```

Reply:
ah okay cool!

Reply:
(It's possible that to get the genericlinux stuff to work here, I'll also need to add a `b.workers.AddExtraWorker(...)` function. but that doesn't exist yet.)